### PR TITLE
referenece Organization Address and Contact Info, not FROM Email Addresses in help SMTP text (lab-879)

### PR DESCRIPTION
**Pull request by michaelmcandrew**
Overview
----------------------------------------
Fairly simple - the help text was wrong and did not match the URL, which is correct. See https://civicrm.stackexchange.com/questions/29261/how-to-setup-civicrm-to-proper-send-authenticated-dkim-emails/29264#29264 and https://lab.civicrm.org/dev/core/issues/879

### CODE REVIEW

**Comment by michaelmcandrew** [comment]
any idea why the conflicts cannot be resolved? seems weird to me.

File: CRM/Admin/Form/Setting/Smtp.php [hunk 1]
```diff
@@ -102,7 +102,7 @@ public function postProcess() {
 
         if (!$domainEmailAddress || $domainEmailAddress == 'info@EXAMPLE.ORG') {
           $fixUrl = CRM_Utils_System::url("civicrm/admin/domain", 'action=update&reset=1');
-          CRM_Core_Error::fatal(ts('The site administrator needs to enter a valid \'FROM Email Address\' in <a href="%1">Administer CiviCRM &raquo; Communications &raquo; FROM Email Addresses</a>. The email address used may need to be a valid mail account with your email service provider.', [1 => $fixUrl]));
+          CRM_Core_Error::fatal(ts('The site administrator needs to enter a valid email address in <a href="%1">Administer CiviCRM &raquo; Communications &raquo; Organization Address and Contact Info</a>. The email address used may need to be a valid mail account with your email service provider.', array(1 => $fixUrl)));
```
Comment (mattwire) [inline CRM/Admin/Form/Setting/Smtp.php]: @michaelmcandrew This fatal should probably be replaced with `CRM_Core_Error::statusBounce` as it's pretty ugly triggering fatal when you get a setting wrong.

**Comment by colemanw** [comment]
@michaelmcandrew could you please rebase out the merge-commit from this PR?

**Comment by yashodha** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
Replaced with https://github.com/civicrm/civicrm-core/pull/14329

**Comment by fkohrt** [comment]
@michaelmcandrew I believe it's the other way round: The text was correct, but linked to the wrong URL. Reason: The SMTP setup checks if a valid FROM address is set up before it allows to send a test email. If no valid FROM address is found, it triggers the message. The organizational contact email address has nothing to do with this and setting it does not help with sending the test email. The correct URL should (probably) be something like `/civicrm/admin/options/from_email_address?reset=1&civicrmDestination=%2Fcivicrm%2Fadmin%2Fconfigtask%3Freset%3D1`.

Honestly it's a problem with the configuration checklist as well as it does not make sense to configure the SMTP setup before the FROM address as the configuration checklists suggests.

**Comment by eileenmcnaughton** [comment]
@fkohrt any chance you can do a PR to fix?

**Comment by fkohrt** [comment]
@eileenmcnaughton done with #15165 :)

But the order in the configuration checklist should be changed as well so that From Address config precedes the SMTP setup.

**Pull request closed without merging**